docs/topics/Threads.md

```diff
@@ -133,35 +136,57 @@ There are four new `GET` routes for enumerating threads in a specific channel:
+Private threads are only synced to you if you are a member or a moderator. Whenever a user is added to a private thread, the Gateway also sends a [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) event. This ensures the client always has a non-null value for that thread.
 
-As mentioned, these lead to a couple of edge cases that are worth going into:
+> info
+> The `THREAD_CREATE` event is also sent when the user is a moderator (and thus would already have the channel in memory).
+
+#### Gaining Access to Public Threads
+
+When a Discord client is added to a public thread but has not yet subscribed to threads, it might not have that public thread in memory yet. However, upon connecting, the Gateway will auto-subscribe apps (which are a type of client) to all thread events and active threads.
 
-### Gaining access to a private thread
+When an app is added to _any_ thread, the Gateway will send it a [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) event.
 
-When a client is added to a private thread, they likely don't have that private thread in memory yet because of the product requirement that we only sync threads you have permission to view. Private threads are only synced to you if you are a member or a moderator. To solve this, whenever a user is added to a private thread, the gateway also sends a [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) dispatch. This ensures the client always has a non-null value for that thread. Note: This is also sent even if the user is a moderator, and thus would already have the channel in memory, mainly for simplicity purposes.
+### Channel Access
 
-### Gaining access to a public thread
+#### Gaining Access to Channels
 
-When a client is added to a public thread, but has not yet subscribed to threads, they might not have that public thread in memory yet. This is actually only a problem for Discord's official clients, and not for bots. The gateway will auto-subscribe bots to all thread dispatches and active threads on connect. But Discord's clients only receive threads that are active and they have also joined on connect in order to reduce the amount of data needed on initial connect. But this means when a user with the official client is added to a thread, that thread now becomes an "active-joined" thread and needs to be synced to the client. To solve this, whenever a user is added to _any_ thread, the gateway also sends a [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) dispatch.
+When an gains access to a channel (for example, they're given the moderator role), they likely won't have the threads in memory for that channel. To account for this, Discord sends a [Thread List Sync](#DOCS_TOPICS_GATEWAY_EVENTS/thread-list-sync) event when this happens.
```

Comment:
Optional, but my 2c is that the little bit in parens before, `(since the gateway only syncs threads that the client has permission to view)` is valuable because it helps explain why this is a problem and connects it back to the principles at the beginning. Some of the extra context like this that we had before I think probably helps with understand why there's all these extra hoops you need to jump through as an app developer. But I don't feel super strongly about it either.